tests/DemosTest.php

```diff
@@ -512,7 +485,7 @@ public function testDemoCallbackError(string $path, string $expectedExceptionMes
         $response = $this->getResponseFromRequest5xx($path);
 
         $this->assertSame(500, $response->getStatusCode());
-        $responseBodyStr = $response->getBody()->getContents();
+        $responseBodyStr = (string) $response->getBody();
```

Comment:
It seems the body content is not rewound somewhere.

__toString() calls rewinds, but we should support `getContents` if possible

Reply:
original `getContents` returns an empty string

I belive it is an issue in the impl, probably the output buffer is not rewound somewhere.

Reply:
`rewind` in in DemosTest added, fixed